modules/client-configuration/pages/clients-sle.adoc

```diff
@@ -0,0 +1,112 @@
+[[clients-sle]]
+= Registering {sle} Clients
+
+This section contains information about registering clients running {sle} operating systems.
+
+// There are probably some support statements that we need to make here. Are SLE clients supported on Uyuni? --LKB 2020-08-12
+
+
+== Add Software Channels
+
+Before you register {sle} clients to your {productname} Server, check that you have the {sle} product enabled, and the required channels are fully synchronized.
+
+The products you need for this procedure are:
+
+ifeval::[{suma-content} == true]
+
+[[opensuse-channels-wizard]]
+[cols="1,1,1,1", options="header"]
+.SLE Products - WebUI
+|===
+
+| OS Version
+| Product Name
+| Updates Channel
+| Pool Channel
+
+| {sles} 15 SP1
+| SUSE Linux Enterprise Server 15 SP1 x86_64
+| SLE-Product-SLES15-SP1-Updates for x86_64
+| SLE-Product-SLES15-SP1-Pool for x86_64
+
+| {sles} 15 SP2
+| SUSE Linux Enterprise Server 15 SP2 x86_64
+| SLE-Product-SLES15-SP2-Updates for x86_64
+| SLE-Product-SLES15-SP2-Pool for x86_64
+
+|===
+
+include::snippets/add_channels_wizard.adoc[]
+
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+
+[[opensuse-channels-cli]]
+[cols="1,1", options="header"]
+.OpenSUSE Channels - CLI
+|===
+
+| OS Version
+| Client Channel
+// Can't find a base channel, is it because it's not supported on Uyuni? --LKB 2020-08-12
```

Comment:
It *is* supported, but you need the same steps you have above, as you get the channels for SLE from SCC

Reply:
> Doc for Uyuni needs a fix.

Yes, it will go into Uyuni as well.

Reply:
Oh, no spacewalk-common-channels option then?

Reply:
Only for the client tools channel. And it's to be executed only after the SLE channels are added from SCC.

However for SUSE manager you don't tell the user to enable the client tools from SCC because they come enabled automatically, so I think you can just have that part common for Uyuni and SUSE Manager. Would be much easier to verify with rendered docs :-D

Reply:
Oh, sorry, I forgot to upload them. I'll update this and upload some for you.